tests/guarded_transmute_pod_vec_permissive/mod.rs

```diff
@@ -2,29 +2,29 @@
 
 
 use self::super::{LeToNative, aligned_vec};
-use safe_transmute::guarded_transmute_pod_vec_permissive;
+use safe_transmute::safe_transmute_vec_permissive;
 
 
 #[test]
 fn too_short() {
-    assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(aligned_vec::<u16>([].as_ref())), Ok(vec![]));
-    assert_eq!(guarded_transmute_pod_vec_permissive::<u16>(aligned_vec::<u16>([0x00].as_ref())), Ok(vec![]));
+    assert_eq!(safe_transmute_vec_permissive::<u16>(aligned_vec::<u16>([].as_ref())), Ok(vec![]));
```

Comment:
Module name doesn't match tested funxion – are we letting this slide 'till the test reorganisation?